.github/workflows/qc.yml

```diff
@@ -20,7 +22,9 @@ jobs:
     # The type of runner that the job will run on
     runs-on: ubuntu-latest
     container: obolibrary/odkfull:v1.4.3
-
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
what does this do?

Reply:
give permission to forked prs to write on it via a gh action